easyaccess/easyaccess.py

```diff
@@ -868,12 +739,12 @@ def query_and_print(self, query, print_time=True, err_arg='No rows selected', su
                     data.index += 1
                     if extra != "":
                         print(colored(extra + '\n', "cyan"))
-                    # ADW: Oracle distinguishes between NaN and Null while pandas 
-                    # does not making this replacement confusing...
-                    # ##try:
-                    # ##    data.fillna('Null', inplace=True)
-                    # ##except:
-                    ###    pass
+                    ## # ADW: Oracle distinguishes between NaN and Null while pandas 
+                    ## # does not making this replacement confusing...
+                    ## try:
+                    ##     data.fillna('Null', inplace=True)
```

Comment:
I know this is old, but sometimes we need to replace Nan or Nulls by a value. Somebody has asked that. We need to keep in mind the average user who might not care whether an entry is a null or Nan but he/she doesn't want it